Sources/Subscription/Managers/SubscriptionManager.swift

```diff
@@ -68,6 +77,21 @@ public final class DefaultSubscriptionManager: SubscriptionManager {
         }
     }
 
+    public var canPurchase: Bool {
+        guard let storePurchaseManager = _storePurchaseManager else { return false }
+
+        switch storePurchaseManager.currentStorefrontRegion {
+        case .usa:
+            return storePurchaseManager.areProductsAvailable
+        case .restOfWorld:
+            if subscriptionFeatureFlagger.isFeatureOn(.isLaunchedROW) || subscriptionFeatureFlagger.isFeatureOn(.isLaunchedROWOverride) {
+                return storePurchaseManager.areProductsAvailable
+            } else {
+                return false
+            }
```

Comment:
Will we also want internal users to be able to bypass this, to allow purchasing via a production build before enabling the feature flag? The override flag would need to be disabled during release week so we should ideally have a workaround mechanism.